src/OpenTelemetry/Metrics/Metric.cs

```diff
@@ -207,6 +207,11 @@ internal static readonly HashSet<(string, string)> DefaultHistogramBoundLongMapp
     /// </summary>
     public string MeterVersion => this.InstrumentIdentity.MeterVersion;
 
+    /// <summary>
+    /// Gets the attributes (tags) for the metric stream.
+    /// </summary>
+    public IEnumerable<KeyValuePair<string, object?>>? MeterTags => this.InstrumentIdentity.MeterTags;
```

Comment:
Only expose this public API only on .NET 8 and above? It's not applicable to the older targets.

Reply:
Ahh never mind! This is coming from DS 8.0.